docs/README.md

```diff
@@ -1,6 +1,7 @@
 # MindsDB Documentation    <a href="https://docs.mindsdb.com?utm_medium=community&utm_source=github&utm_campaign=mindsdb%20repo"><img src="https://img.shields.io/website?url=https%3A%2F%2Fwww.mindsdb.com%2F" alt="MindsDB Docs"></a>	
 
 ## Running the docs locally
+Run the following commands in your local repo, and make sure you are located at 'docs' folder.
```

Comment:
Please remove all the content from the `Running the docs locally` chapter and replace it with the following:

> \## Running the Docs Locally
> 
> \<Info>
>     \*\*Prerequisite\*\*
>     You should have installed Git (version 2.30.1 or higher) and Node.js (version 18.10.0 or higher).
> \</Info>
> 
> Step 1. Clone the MindsDB Git repository:
> 
> \```console
> git clone https://github.com/mindsdb/mindsdb.git
> \```
> 
> Step 2. Install Mintlify on your OS:
> 
> \```console
> npm i mintlify -g
> \```
> 
> Step 3. Go to the \`docs\` folder inside the cloned MindsDB Git repository and start Mintlify there:
> 
> \```console
> mintlify dev
> \```
> 
> The documentation website is now available at \`http://localhost:3000\`.
> 
> \<Warning>
>   \*\*Getting an Error?\*\*
>   If you use the Windows operating system, you may get an error saying \`no such file or directory: C:/Users/Username/.mintlify/mint/client\`. Here are the steps to troubleshoot it:
>         - Go to the \`C:/Users/Username/.mintlify/\` directory.
>         - Remove the \`mint\` folder.
>         - Open the Git Bash in this location and run \`git clone https://github.com/mintlify/mint.git\`.
>         - Repeat step 3.
> \</Warning>